pyproject.toml

```diff
@@ -73,6 +73,7 @@ pytest-cov = "^2.12.1"
 pytest-env = "^0.6.2"
 # benchmark
 asv = "^0.4.2"
+virtualenv = "^20.13.2"
```

Comment:
hu? why do we have an explict dependency to virtualenv (inside poetry)?

Reply:
Because of *airspeed velocity* (that of course I want to run inside `poetry`).

I know there are plenty of things to be done, but I need to measure the improvement in memory footprint, so my idea was to set up a couple of quick benchmarks:
- compute non-TMC observable: nothing should change
- compute TMC observable: it should pass from a much larger memory consumption than the former, to a comparable one

Reply:
I see, so when it measures it needs an env ...